.gitignore

```diff
@@ -1,2 +1,12 @@
-.env*
-.idea
+__pycache__/
+*.py[cod]
+*$py.class
```

Comment:
Perhaps let's merge .gitignore first so everyone can stop doing this ha

Reply:
Yep. The big .gitignore change should be done, I just need to pick it up and merge here.